skypy/halo/tests/test_quenching.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from scipy import stats
+from collections import Counter
+
+from skypy.halo.quenching import environment_quenched, mass_quenched
+
+
+def test_environment_quenched():
+    # Test the quenching process follows a binomial distribution
+    n, p = 1000, 0.7
+    quenched = environment_quenched(n, p)
+    number_quenched = Counter(quenched)[0]
+
+    p_value = stats.binom_test(number_quenched, n=n, p=p,
+                               alternative='greater')
```

Comment:
```suggestion
    p_value = stats.binom_test(number_quenched, n=n, p=p)
```
I think we want to test the default `alternative='two-sided'` i.e. check that the probability is neither great or less than p

Reply:
It does not pass the test if `alternative='two-sided'`. I will look into this more carefully but I thought `alternative='greater'` was doing the right thing. Any comments?

Reply:
I don't think this is why the test is failing; see my suggestion r.e. `Counter` and then try the default `alternative='two-sided'` again.